### PR TITLE
Improve CC gallery example

### PR DESCRIPTION
**Pull request by JoOkuma**

### CODE REVIEW

**Comment by ziw-liu** [comment]
You would need to change the file name to `run_*.py` if this needs to be executed.

(Not that it's going to work since we don't have napari installed.)

**Comment by ziw-liu** [comment]
Maybe do this?

```py
try:
    import napari
except ImportError:
    sys.exit(0)
viewer = ...
napari.run()
```

File: docs/source/conf.py [hunk 1]
```diff
@@ -209,7 +210,7 @@
     "examples_dirs": "../examples",
     "gallery_dirs": "auto_examples",
     "download_all_examples": False,
-    "filename_pattern": "/run_",
+    # "filename_pattern": "/run_",
```
Comment (ziw-liu) [inline docs/source/conf.py:213]: The default is `'/plot'`, not everything.

**Comment by JoOkuma** [comment]
@ziw-liu 
> (Not that it's going to work since we don't have napari installed.)
It's working with `sphinx-gallery` in an environment without napari, it just can't be called as a `__main__` file.

**Comment by ziw-liu** [comment]
> @ziw-liu
> 
> > (Not that it's going to work since we don't have napari installed.)
> 
> It's working with `sphinx-gallery` in an environment without napari, it just can't be called as a `__main__` file.

Rendering will always work (and worked before this PR). My point was that the blocks before importing napari could still be executed (like the first blocks in the large array example) so that the output of loading times (L43 and L50) can be displayed.

File: docs/source/conf.py (hunk 1 above)
Comment (ziw-liu) [reply docs/source/conf.py:213]: Will need to undo this change before merging.

**Review by ziw-liu — APPROVED** [review]
This won't be run in CI for now. If the filename is changed to `run_view_clearcontrol_dataset.py` it will run and the last block should error like [this](https://sphinx-gallery.github.io/stable/auto_examples/no_output/plot_raise.html), which is probably fine.

**Pull request merged**